ui/public/updateBrowser.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="./favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="description" content="TiDB Dashboard" />
+    <title>TiDB Dashboard</title>
+    <style>
+      body {
+        font-family: sans-serif;
+      }
+      #root {
+        margin: 0;
+        padding-left: 24px;
+      }
+      ul.logos {
+        padding: 0;
+        padding-top: 8px;
+      }
+      ul.logos li {
+        float: left;
+        list-style: none;
+        padding: 12px;
+        background-color: #f5f5f5;
+        margin-right: 16px;
+      }
+      .logos a {
+        display: block;
+        width: 110px;
+        background-repeat: no-repeat;
+        background-size: 110px auto;
+        padding-top: 120px;
+        text-align: center;
+        text-decoration: none;
+        color: inherit;
+      }
+      .chrome {
+        background-image: url(./assets/img/chrome.png);
+      }
+      .edge {
+        background-image: url(./assets/img/edge.png);
+      }
+      .firefox {
+        background-image: url(./assets/img/firefox.png);
+      }
+      .safari {
+        background-image: url(./assets/img/safari.png);
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root">
+      <h1 id="title">Please update your browser</h1>
+      <p id="desc">
+        Your browser isn't supported anymore. Update it to get the best TiDB
```

Comment:
Notice that YouTube will display different messages when current browser is up-to-date while browsing the same page:
![image](https://user-images.githubusercontent.com/1916485/84763799-58d38580-afff-11ea-866c-c73069056063.png)

This avoid confusing when user switched the browser, and copy-paste the URL.